soroban-builtin-sdk-macros/Cargo.toml

```diff
@@ -15,7 +15,7 @@ proc-macro = true
 [dependencies]
 syn = {version="=2.0.39",features=["full"]}
 quote = "=1.0.33"
-proc-macro2 = "=1.0.69"
+proc-macro2 = "1.0.78"
```

Comment:
It would be enough to remove the = while keeping the number the number the same. Changing the number changes the min version required which I don't think we could do as it would force stellar-core to use a newer version of the crate.